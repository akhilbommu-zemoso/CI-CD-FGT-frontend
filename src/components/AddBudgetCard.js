import React from 'react';
import { makeStyles } from '@material-ui/core/styles';
import { FTCard } from '../atoms/FTCard';
const useStyles = makeStyles({
    cardStyle: {
        width: '200px',
        fontWeight: 900,
        borderRadius: '6px',
        border: '1px,solid,black',
        color: 'rgb(22,101,216)',
        backgroundColor: 'rgb(255, 255, 255)',
        textAlign: 'center',
        '&:hover': {
            backgroundColor: "rgb(4,136,255)",
            color: 'white'
        }
    },
    content: {
        marginTop: "14px",
        fontSize: 12,
        textAlign: 'center',
        fontWeight: 500,
        color: 'black',
        textAlign: 'center',
    },
});
export default function Budgetcard(props) {
    const classes = useStyles();
    return (
        <FTCard
            cardStyle={classes.cardStyle}
            contentStyle={classes.content}
            headerStyle={classes.header}
            header={"Create your budget"}
            content={"Take control on your expenses by creating a budget on your spendings"}
        />
    );
}