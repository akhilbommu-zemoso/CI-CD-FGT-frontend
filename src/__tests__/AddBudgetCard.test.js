import React, { Component } from 'react'
import { render } from '@testing-library/react'
import AddBudgetCard from '../components/AddBudgetCard'

it('should take a snapshot', () => {
    const { asFragment} = render(<AddBudgetCard/>)
    expect(asFragment(<AddBudgetCard />)).toMatchSnapshot()
});

it('check for header', () => {
    const { getByText } = render(<AddBudgetCard />)
    expect(getByText("Create your budget")).toBe
});

it('check for content', () => {
    const { getByText } = render(<AddBudgetCard />)
    expect(getByText("Take control on your expenses by creating a budget on your spendings")).toBe
});
