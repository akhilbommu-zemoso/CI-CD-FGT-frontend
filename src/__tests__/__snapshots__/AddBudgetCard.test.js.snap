// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiPaper-root MuiCard-root makeStyles-cardStyle-1 MuiPaper-elevation1 MuiPaper-rounded"
  >
    <div
      class="MuiCardContent-root"
    >
      <p
        class="MuiTypography-root MuiTypography-body1"
      >
        Create your budget
      </p>
      <p
        class="MuiTypography-root makeStyles-content-2 MuiTypography-body1"
      >
        Take control on your expenses by creating a budget on your spendings
      </p>
      <p
        class="MuiTypography-root MuiTypography-body1"
      />
    </div>
  </div>
</DocumentFragment>
`;
